docs(themeing): Move all themes in separate files (#1782)
IgniteUI/igniteui-docfx: en/components/themes/themes-list/material.md

### Material theme

> Material theme is the default theme for igniteUI for Angular

Reviewer: ```suggestion
The Material theme is the default theme for igniteUI for Angular.
```

> in order to use it, include one of the following mixins:

Reviewer: ```suggestion
In order to use it, include one of the following mixins:
```

> ```scss

Reviewer: ```suggestion
### Default Theme

```scss
```

@include igx-theme($default-palette);
```

> 

Reviewer: ```suggestion
### Material Light Theme
```

> For the `light` version of the `Material theme` you can use the following mixin: 

Reviewer: ```suggestion
For the `light` version of the Material theme, you can use the following mixin: 
```

```scss
@include igx-light-theme($default-palette);
```

> 

Reviewer: ```suggestion
### Material Dark Theme

```

> For the `Dark` version of the `Material theme` you can use the following mixin: 

Reviewer: ```suggestion
For the `Dark` version of the Material theme, you can use the following mixin: 
```

```scss
@include igx-dark-theme($default-palette);
```

> Both `igx-light-theme` and `igx-dark-theme` mixins extend the `igx-theme` mixin.

Reviewer: ```suggestion
Both the `igx-light-theme` and the `igx-dark-theme` mixins extend the `igx-theme` mixin.
```

> We create those mixins just for your convenience, otherwise, if you want to use the `igx-theme` but with a dark palette and dark schema you have to manually do the following:

Reviewer: ```suggestion
We created those wrapper mixins just for your convenience, otherwise, if you want to use the `igx-theme` but with a dark palette and dark schema you have to do the following:
```

```scss
$primary-color: #2ab759;
$secondary-color: #f96a88;

$my-color-palette: igx-palette(
    $primary: $primary-color,
    $secondary: $secondary-color,
    // This will ensure that all the black text, borders, etc  will 

>     // become some shade of white in order to be visible in the dark theme

Reviewer: ```suggestion
    // become some shade of white in order to be visible against dark backgrounds
```

    $grays: #fff 
    // If you don't specify the rest of the colors like $info and $error they will use their default values from the $default-palette 
);

$my-palette: igx-palette(
  $primary: #09f,
  $secondary: #ff4ba4,

> );

Reviewer: I believe this declaration is not needed in this code sample anymore.

## Default Theme

> Material theme is the default theme for igniteUI for Angular. To use it include `igx-theme()` mixins after the igx-core.  

Reviewer: ```suggestion
The Material theme is the default theme in Ignite UI for Angular. To use it include the `igx-theme` mixins after `igx-core`.  
```

```scss
@import '~igniteui-angular/lib/core/styles/themes/index';
@include igc-core();
@include igx-theme($default-palette);
```

### Material Light Theme
```scss
@import '~igniteui-angular/lib/core/styles/themes/index';
@include igc-core();
@include igx-light-theme($default-palette);
```

### Material Dark Theme
```scss
@import '~igniteui-angular/lib/core/styles/themes/index';
@include igc-core();
@include igx-dark-theme($default-palette);
```

Both `igx-light-theme` and `igx-dark-theme` mixins includes the `igx-theme` mixin.
We create those mixins just for your convenience, otherwise, if you want to use the `igx-theme` but with a dark palette and dark schema you have to manually do the following:

```scss
@import '~igniteui-angular/lib/core/styles/themes/index';
@include igc-core();

$primary-color: #2ab759;
$secondary-color: #f96a88;

$my-color-palette: igx-palette(
    // IMPORTANT, the primary and secondary colors are required
    $primary: $primary-color,
    $secondary: $secondary-color,

>     // The $grays will ensure that all the black text, borders, etc  will 

Reviewer: ```suggestion
    // The $grays will ensure that all the black text, borders, etc will 
```

>     // become a shade of whites in order to be visible in the dark theme

Reviewer: ```suggestion
    // become a shade of white in order to be visible against a dark background
```
